Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -363,6 +363,22 @@ TEST_CASE(test_png)
     TRY_OR_FAIL(expect_single_frame(*plugin_decoder));
 }
 
+TEST_CASE(test_apng)
+{
+    auto file = TRY_OR_FAIL(Core::MappedFile::map(TEST_INPUT("png/apng-1-frame.png"sv)));
```

Comment:
I suppose this is `062.png` from `fDAT-inherits-cICP.html`, it would be nice to say where you got this file in the commit message.